app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for person in people:
+        Person(person["name"], person["age"])
```

Comment:
here you can do a list comprehension and save it to some variable such a list_of_persons or result and simply return it, cause now you did two actions instead of 1.

```
result = [Person(person["name"], person["age"]) for person in people]
....
....
your_code
....
....
return result
```